Sources/PackageCollectionsSigning/Certificate/CertificatePolicy.swift

```diff
@@ -346,15 +346,17 @@ private struct BoringSSLOCSPClient {
                         return
                     }
 
-                    let response = d2i_OCSP_RESPONSE_bio(bio, nil)
+                    guard let response = d2i_OCSP_RESPONSE_bio(bio, nil),
+                          let basicResp = OCSP_response_get1_basic(response) else {
```

Comment:
This has introduced a memory leak on the failure path if `response` is non-nil` but `basicResp` is `nil`.

Reply:
Amended https://github.com/apple/swift-package-manager/pull/3859/commits/a1c79c42564fdad57f2ccb2e91d246d6ad289dcb